intel-retail/automated-self-checkout
# feat: implement pipeline profile for opencv-ovms runs

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- if haven't built the model-server yet, the first time you can build it with `make build-ovms-server` and it will take a while.
- run camera simulator: `./camera-simulator/camera-simulator.sh`
- build ovms-client: `make build-ovms-client`
- docker-run with PIPELINE_PROFILE env set to "grpc_python":  `PIPELINE_PROFILE="grpc_python" sudo -E ./docker-run.sh --workload opencv-ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0`
- exam the ovms-client docker container log and should see no error:  `docker logs ovms-client0`
- exam the result outputs and should see the pipeline fps output: `sudo cat ./results/r0.jsonl`
- 
## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
